src/decode-events.c

```diff
@@ -566,6 +566,10 @@ const struct DecodeEvents_ DEvents[] = {
             "decoder.nsh.unknown_payload",
             NSH_UNKNOWN_PAYLOAD,
     },
+    {
+            "decoder.too_many_layers",
```

Comment:
we also need a rule for this in `decoder-events.rules`

Reply:
ok